src/mixer/audio.ts

```diff
@@ -449,6 +448,18 @@ export class AudioEngine extends ((EngineEmitter as unknown) as {
     this.micMixGain.gain.value = value;
   }
 
+  setMicProcessingEnabled(value: boolean) {
+    // Disconnect whatever was connected before.
```

Comment:
Potentially worth checking what it was before, if it's worth even doing anything? I'm slightly concerned this may cause connection thrashing, but I'm not sure how relevant that is.

Reply:
It's either connected to micCompressor or micMixGain (depending on if we're going from enabled to disabled or vice-versa). (and the micPrecompAnalyser). But afaict you can't disconnect only one thing (without destroying it I guess), so you have to disconnect all anyway.

Reply:
Added in comment.